pkgs/development/beam-modules/fetch-mix-deps.nix

```diff
@@ -0,0 +1,44 @@
+{ stdenvNoCC, elixir, hex, rebar, rebar3, cacert, git }:
+
+{ name, version, sha256, src, mixEnv ? "prod", debug ? false, meta ? { } }:
+
+with stdenvNoCC.lib;
+
+stdenvNoCC.mkDerivation ({
+  name = "mix-deps-${name}-${version}";
+
+  phases = [ "configurePhase" "downloadPhase" ];
```

Comment:
```suggestion
```

Please do not overwrite phases. Disable the ones you do not want to run with ``dontBuild = true;``. downloadPhase is a non standard phase, right? I am not sure if it is allowed in such context.

Reply:
I also took it from fetch rebar deps https://github.com/NixOS/nixpkgs/blob/master/pkgs/development/beam-modules/fetch-rebar-deps.nix
But I thought it was weird as well.